src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
Replace `combineSlices` with `combineReducers` from `@reduxjs/toolkit`. `combineReducers` is the correct function to use when combining multiple reducers into a single root reducer.